src/Symfony/Component/Notifier/Bridge/Twitter/TwitterTransport.php

```diff
@@ -0,0 +1,297 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Twitter;
+
+use Symfony\Component\Mime\Part\DataPart;
+use Symfony\Component\Mime\Part\Multipart\FormDataPart;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\ChunkInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+final class TwitterTransport extends AbstractTransport
+{
+    protected const HOST = 'api.twitter.com';
+
+    private static $nonce;
+
+    private string $apiKey;
+    private string $apiSecret;
+    private string $accessToken;
+    private string $accessSecret;
+
+    public function __construct(string $apiKey, #[\SensitiveParameter] string $apiSecret, string $accessToken, #[\SensitiveParameter] string $accessSecret, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
```

Comment:
Should we mark the access token sensitive too ?

Reply:
And apikey too?

Because none of them are used in toString as well

Reply:
OK